pyro/infer/mcmc/mcmc.py

```diff
@@ -24,20 +24,23 @@ class MCMC(TracePosterior):
         during the warmup phase are discarded.
     """
 
-    def __init__(self, kernel, num_samples, warmup_steps=0):
+    def __init__(self, kernel, num_samples, warmup_steps=0, lag_steps=1):
```

Comment:
Please describe `lag_steps` in the above docstring. (I think I've also seen this parameter named `sample_every`)